app-arch/bzip2/bzip2-1.0.6-r10.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic multilib-minimal toolchain-funcs
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SUSE_PV="${PV}.0.1"
```

Comment:
Why are we using 0.1 and not 0.2?

Reply:
Huh, buecause I wasn't aware of it at the time. Lemme look into that, then.